cobbler/actions/buildiso/standalone.py

```diff
@@ -164,7 +177,7 @@ def _generate_item(
         cfg_parts.isolinux.append(isolinux)
         cfg_parts.grub.append(grub)
         cfg_parts.bootfiles_copysets.append(to_copy)
-        autoinstall_data[name] = Autoinstall(autoinstall, repos)
+        autoinstall_data[name] = Autoinstall(autoinstall, cast(List[str], repos))
```

Comment:
[nit] I'm still not a fan of using `cast()`, is this required if we add type hints to the variable declaration?

Reply:
Indeed, setting type during variable declaration helps here so we can avoid the usage of `cast()`. 

Thanks for the suggestion! :+1: